src/features/XIT/REP/REP.ts

```diff
@@ -1,10 +1,22 @@
 import REP from '@src/features/XIT/REP/REP.vue';
+import { getEntityNaturalIdFromAddress } from '@src/infrastructure/prun-api/data/addresses';
+import { getParameterSites } from './entries';
 
 xit.add({
   command: ['REP', 'REPAIR', 'REPAIRS'],
   name: 'REPAIRS',
   description: 'Shows the materials to repair buildings.',
   optionalParameters: 'Planet Identifier(s)',
-  contextItems: () => [{ cmd: 'BRA' }],
+  contextItems: (parameters: string[]) => {
```

Comment:
```suggestion
  contextItems: parameters => {
```
Code style: don't add type definitions where tsc (TypeScript compiler) is able to derive the type automatically